microservices/front-end/src/components/QuestionBank.css

```diff
@@ -0,0 +1,107 @@
+/* set button component */
+button {
+    border-radius: 8px;
+    border: 1px solid transparent;
+    padding: 0.6em 1.2em;
+    font-size: 1em;
+    font-weight: 500;
+    font-family: inherit;
+    background-color: #f9f9f9;
+    cursor: pointer;
+    transition: border-color 0.25s;
+    margin: 3px;
+}
+
+button:hover {
+    border-color: #646cff;
+    
+}
+button:focus,
+button:focus-visible {
+    outline: 4px auto -webkit-focus-ring-color;
+}
+
+.action-button {
+    border: 2px solid #A09797;
+}
+.action-button:hover {
+    border-color: #A09797;
+    background-color: #E6E1E1;
+}
+
+/* Align text center for category, and complexity */
+.center-align-cell {
+    text-align: center;
+}
+  
+/* Style the header row of the table */
+/* th {
+    font-size: 1.25em;
+    background-color: #C4C5C5;
+    text-align: center;
+    padding: 10px;
+} */
+
```

Comment:
Let's remove unused code to keep the file clean
```suggestion
```

Reply:
Ok removed